### PR TITLE
Enable AppVeyor CI for running test suite on Windows env

### LINKED ISSUE #1684: Test Windows builds with Appveyor integration
Just like scrapy has Travis CI integration, scrapy can also be tested on Windows with http://www.appveyor.com/

### PR DESCRIPTION
**Pull request by dangra**
Test windows builds on Appveyor CI. 
Closes #1684

### CODE REVIEW

**Comment by kmike** [comment]
A related pull request: https://github.com/scrapy/scrapy/pull/1756, though I hope many of the workarounds from that PR are not needed now.

**Comment by dangra** [comment]
all green @kmike ! ready to review and get it merged for 1.6 release 👯

File: tests/test_feedexport.py [hunk 1]
```diff
@@ -238,11 +240,19 @@ def run_and_export(self, spider_cls, settings=None):
...
-            with open(res_name, 'rb') as f:
-                defer.returnValue(f.read())
+            with open(res_path, 'rb') as f:
+                content = f.read()
 
         finally:
-            shutil.rmtree(tmpdir)
+            # FIXME: Windows fails to remove the file because FeedExporter 
+            # keeps a reference to the temporal file even after
+            # the spider finished.
```
Comment (kmike) [inline tests/test_feedexport.py]: Should we open an issue for that?

File: tests/test_feedexport.py [hunk 2]
```diff
@@ -238,11 +240,19 @@ def run_and_export(self, spider_cls, settings=None):
...
+            try:
+                shutil.rmtree(tmpdir)
```
Comment (kmike) [inline tests/test_feedexport.py]: can we use `rmtree(tmpdir, ignore_errors=True)`?

Comment (dangra) [reply tests/test_feedexport.py]: Good idea. fixing.

File: tests/test_feedexport.py (hunk 1 above)
Comment (dangra) [reply tests/test_feedexport.py]: +1

Comment (dangra) [reply tests/test_feedexport.py]: https://github.com/scrapy/scrapy/issues/3391

**Comment by kmike** [comment]
Awesome, thanks @dangra for picking this up and @jdemaeyer for the initial work!

**Pull request merged**